assets/scss/app.scss

```diff
@@ -164,3 +164,22 @@ ul {
   margin-top: 0.5em;
   margin-bottom: 2em;
 }
+
+#tenant-additional-details-group h2 {
+  font-family: "nta", Arial, sans-serif;
+  font-weight: 700;
+  text-transform: none;
+  font-size: 16px;
+  line-height: 1.25;
+  margin-top: 0;
+  margin-bottom: 0;
+  @media (min-width: 641px) {
+      font-size: 19px;
+      line-height: 1.31579;
+      margin-top: 0;
+  }
```

Comment:
This seems like overly specific css - the styles shouldn't be *only* relevant to this field, and in particular, shouldn't be dependent on the field name configuration.